src/components/EventDetails.tsx

```diff
@@ -0,0 +1,84 @@
+import { ReactElement } from "react";
+import Frame from ".././assets/Frame.svg";
+import LinkSVG from ".././assets/Link.svg";
+import MeatballMenuSVG from ".././assets/MeatballMenu.svg";
+import PinSVG from ".././assets/Pin.svg";
+import TextIconSVG from ".././assets/TextIcon.svg";
+import { Event } from "./EventCard";
+
+export type EventDetailsProps = {
+  event: Event;
+};
+
+const EventDetails = ( {event}:EventDetailsProps): ReactElement => {
+  return (
+    <div
+      className="flex h-screen w-screen flex-col p-10"
+      style={{ gap: "2rem" }}
+    >
+      <div className="flex flex-row items-center" style={{ gap: "1rem" }}>
+        <img src={Frame} alt="Back arrow" className="p-1 pt-0" />
```

Comment:
Don't think you need the padding here much for the icons, correct me if i'm wrong doe